sendKeys / elementIdKeys does not work

I get this Java exception whenever trying to use client.sendKeys. I'm using `selenium-server-standalone-2.31.0.jar`

```
[17:47:36]:  COMMAND    POST     "/wd/hub/session/285cd63e-3a9c-4e99-b8f1-df626dbba187/element"
[17:47:36]:  DATA                {"using":"css selector","value":"#combobox"}
17:47:36.115 INFO - Executing: [find element: By.selector: #combobox] at URL: /session/285cd63e-3a9c-4e99-b8f1-df626dbba187/elemen
t)
17:47:36.126 INFO - Done: /session/285cd63e-3a9c-4e99-b8f1-df626dbba187/element
[17:47:36]:  RESULT              {"ELEMENT":"7"}
[17:47:36]:  COMMAND    POST     "/wd/hub/session/285cd63e-3a9c-4e99-b8f1-df626dbba187/element/7/value"
[17:47:36]:  DATA                {"value":"hello"}
17:47:36.129 WARN - Fatal, unhandled exception: /session/285cd63e-3a9c-4e99-b8f1-df626dbba187/element/7/value: java.lang.ClassCast
Exception: java.lang.String cannot be cast to java.util.List
17:47:36.131 INFO - WebDriver remote server: Fatal, unhandled exception: /session/285cd63e-3a9c-4e99-b8f1-df626dbba187/element/7/v
alue: org.openqa.selenium.WebDriverException: java.lang.ClassCastException: java.lang.String cannot be cast to java.util.List
Build info: version: '2.31.0', revision: '1bd294d', time: '2013-02-27 20:52:59'
System info: os.name: 'Linux', os.arch: 'i386', os.version: '3.8.0-20-generic', java.version: '1.7.0_21'
Driver info: driver.version: unknown
17:47:36.131 WARN - EXCEPTION 
javax.servlet.ServletException: org.openqa.selenium.WebDriverException: java.lang.ClassCastException: java.lang.String cannot be c
ast to java.util.List
Build info: version: '2.31.0', revision: '1bd294d', time: '2013-02-27 20:52:59'
System info: os.name: 'Linux', os.arch: 'i386', os.version: '3.8.0-20-generic', java.version: '1.7.0_21'
Driver info: driver.version: unknown
        at org.openqa.selenium.remote.server.DriverServlet.handleRequest(DriverServlet.java:206)
        at org.openqa.selenium.remote.server.DriverServlet.doPost(DriverServlet.java:167)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:727)
        at org.openqa.selenium.remote.server.DriverServlet.service(DriverServlet.java:139)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
        at org.openqa.jetty.jetty.servlet.ServletHolder.handle(ServletHolder.java:428)
        at org.openqa.jetty.jetty.servlet.ServletHandler.dispatch(ServletHandler.java:677)
        at org.openqa.jetty.jetty.servlet.ServletHandler.handle(ServletHandler.java:568)
        at org.openqa.jetty.http.HttpContext.handle(HttpContext.java:1526)
        at org.openqa.jetty.http.HttpContext.handle(HttpContext.java:1479)
        at org.openqa.jetty.http.HttpServer.service(HttpServer.java:914)
        at org.openqa.jetty.http.HttpConnection.service(HttpConnection.java:820)
        at org.openqa.jetty.http.HttpConnection.handleNext(HttpConnection.java:986)
        at org.openqa.jetty.http.HttpConnection.handle(HttpConnection.java:837)
        at org.openqa.jetty.http.SocketListener.handleConnection(SocketListener.java:243)
        at org.openqa.jetty.util.ThreadedServer.handle(ThreadedServer.java:357)
        at org.openqa.jetty.util.ThreadPool$PoolThread.run(ThreadPool.java:534)
Caused by: org.openqa.selenium.WebDriverException: java.lang.ClassCastException: java.lang.String cannot be cast to java.util.List
Build info: version: '2.31.0', revision: '1bd294d', time: '2013-02-27 20:52:59'
System info: os.name: 'Linux', os.arch: 'i386', os.version: '3.8.0-20-generic', java.version: '1.7.0_21'
Driver info: driver.version: unknown
        at org.openqa.selenium.remote.server.JsonHttpRemoteConfig.handleRequest(JsonHttpRemoteConfig.java:199)
        at org.openqa.selenium.remote.server.DriverServlet.handleRequest(DriverServlet.java:201)
        ... 16 more
Caused by: java.lang.ClassCastException: java.lang.String cannot be cast to java.util.List
        at org.openqa.selenium.remote.server.handler.SendKeys.setJsonParameters(SendKeys.java:39)
        at org.openqa.selenium.remote.server.rest.ResultConfig.setJsonParameters(ResultConfig.java:301)
        at org.openqa.selenium.remote.server.rest.ResultConfig.handle(ResultConfig.java:191)
        at org.openqa.selenium.remote.server.JsonHttpRemoteConfig.handleRequest(JsonHttpRemoteConfig.java:192)
        ... 17 more
17:47:36.133 ERROR - /wd/hub/session/285cd63e-3a9c-4e99-b8f1-df626dbba187/element/7/value: 
org.openqa.selenium.WebDriverException: java.lang.ClassCastException: java.lang.String cannot be cast to java.util.List
Build info: version: '2.31.0', revision: '1bd294d', time: '2013-02-27 20:52:59'
System info: os.name: 'Linux', os.arch: 'i386', os.version: '3.8.0-20-generic', java.version: '1.7.0_21'
Driver info: driver.version: unknown
        at org.openqa.selenium.remote.server.JsonHttpRemoteConfig.handleRequest(JsonHttpRemoteConfig.java:199)
        at org.openqa.selenium.remote.server.DriverServlet.handleRequest(DriverServlet.java:201)
        at org.openqa.selenium.remote.server.DriverServlet.doPost(DriverServlet.java:167)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:727)
        at org.openqa.selenium.remote.server.DriverServlet.service(DriverServlet.java:139)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
        at org.openqa.jetty.jetty.servlet.ServletHolder.handle(ServletHolder.java:428)
        at org.openqa.jetty.jetty.servlet.ServletHandler.dispatch(ServletHandler.java:677)
        at org.openqa.jetty.jetty.servlet.ServletHandler.handle(ServletHandler.java:568)
        at org.openqa.jetty.http.HttpContext.handle(HttpContext.java:1526)
        at org.openqa.jetty.http.HttpContext.handle(HttpContext.java:1479)
        at org.openqa.jetty.http.HttpServer.service(HttpServer.java:914)
        at org.openqa.jetty.http.HttpConnection.service(HttpConnection.java:820)
        at org.openqa.jetty.http.HttpConnection.handleNext(HttpConnection.java:986)
        at org.openqa.jetty.http.HttpConnection.handle(HttpConnection.java:837)
        at org.openqa.jetty.http.SocketListener.handleConnection(SocketListener.java:243)
        at org.openqa.jetty.util.ThreadedServer.handle(ThreadedServer.java:357)
        at org.openqa.jetty.util.ThreadPool$PoolThread.run(ThreadPool.java:534)
Caused by: java.lang.ClassCastException: java.lang.String cannot be cast to java.util.List
        at org.openqa.selenium.remote.server.handler.SendKeys.setJsonParameters(SendKeys.java:39)
        at org.openqa.selenium.remote.server.rest.ResultConfig.setJsonParameters(ResultConfig.java:301)
        at org.openqa.selenium.remote.server.rest.ResultConfig.handle(ResultConfig.java:191)
        at org.openqa.selenium.remote.server.JsonHttpRemoteConfig.handleRequest(JsonHttpRemoteConfig.java:192)
        ... 17 more
```

--- a/lib/protocol/elementIdKeys.js
+++ b/lib/protocol/elementIdKeys.js
@@ -7,6 +7,6 @@ exports.command = function (id, keys, callback) {
 
     requestOptions.path = requestOptions.path.replace(/:id/gi, id);
 
-    this.requestHandler.create(requestOptions,{"value":keys},callback);
+    this.requestHandler.create(requestOptions,{"value":[keys]},callback);
 
 };

